plugins/cpp/service/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ include_directories(
   ${PROJECT_SOURCE_DIR}/util/include
   ${PROJECT_SOURCE_DIR}/webserver/include
   ${PROJECT_SOURCE_DIR}/model/include
+  ${PROJECT_SOURCE_DIR}/parser/include
```

Comment:
Thank you for your remark and explanation, I was not completely aware of this though you might see that this dependency was not used in the final version and was left in by accident.
My only request is that in the future, please scatter your comments in an acceptable manner since as you mentioned, this is a professional environment and communicating in memes might come across as amateur or hencedown rude.